content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -236,12 +280,12 @@ password.
 Multicurrency
 ~~~~~~~~~~~~~
 
-The official currency exchange rate in Peru is provided by the Bank of Peru. Odoo can connect
-directly to its services and get the currency rate either automatically or manually.
+The official currency exchange rate in Peru is provided by the SUNAT. Odoo can connect directly to
+its services and get the currency rate either automatically or manually.
 
-.. image:: peru/peru-multicurrency.png
+.. image:: peru/l10n-pe-banksync-sunat.png
    :align: center
```

Comment:
Our content guidelines have been updated and we no longer add any alignment to screenshots